Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
I lazily took advantage of the extended scope.  I'll have to sort out passing the correct methods and variable to get them to work.  I'll peek at it tomorrow.